src/Humanizer.Tests/ApiApprover/PublicApiApprovalTest.approve_public_api.approved.txt

```diff
@@ -1028,12 +1030,12 @@ namespace Humanizer.DateTimeHumanizeStrategy
     }
     public class PrecisionDateTimeHumanizeStrategy : Humanizer.DateTimeHumanizeStrategy.IDateTimeHumanizeStrategy
     {
-        public PrecisionDateTimeHumanizeStrategy(double precision = 0.75) { }
+        public PrecisionDateTimeHumanizeStrategy(double precision = 0,75) { }
```

Comment:
Done. The PR is still [pending] (https://github.com/JakeGinnivan/ApiApprover/pull/13).